test/TestUserAcceptance.hs

```diff
@@ -835,7 +850,7 @@ startApplication = do
   sessionName <- getSessionName
   -- add $STACK_ARGS so that we use the same resolver as was used for the build
   let cmdline = "stack $STACK_ARGS exec purebred -- --database " <> testmdir <> "\r"
-  liftIO $ callProcess "tmux" $
+  liftIO $ runProcess_ $ proc "tmux" $
```

Comment:
(nit) I don't think we need the `liftIO` anymore (also in a bunch of other places).  Don't bother fixing now, unless there's something else to do.

Reply:
Ah good point actually. I thought I saw it somewhere else too. I'll try to keep it in mind for the next patch set, since I want to try out if I can make the tests more reliable when it comes to figuring out if the UI has been repainted.